src/components/EmptyState/meta.json

```
@@ -25,6 +25,12 @@
         "type": "function",
         "default": "`console.log('You clicked the empty state button!')`",
         "description": "The function used to define an action when the call to action button is clicked."
+      },
+      {
+        "name": "buttonUrl",
+        "type": "string",
+        "default": "`https://google.com`",
```

@MattIGolden [It looks like](https://github.com/MattIGolden/nr1-community/blob/b95a741c3fe20851cf69006336ce97416814f4d8/src/components/EmptyState/index.js#L53) the default for the `buttonUrl` prop is actually empty. The default for the code example in the the demo is "https://google.com/" but when someone installs the component and uses it in their project it will be empty. In other words if you use this component in a project and don't provide and don't use the `buttonUrl` prop, what will happen? Nothing.

Can you change the "default" value from google to an empty string ("")?
```suggestion
        "default": "",
```